tests/data/zuericrop/data.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import h5py
+
+import numpy as np
+
+
+# Sentinel-2 is 12-bit with range 0-4095
+SENTINEL2_MAX = 4095

```

Comment:
I think this should be 4096, np.random.randint input is "one above the largest integer"